demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -408,6 +408,12 @@ def are_valid_files(self) -> str:
             self.validate_pack_meta_file()
         # We only check pack dependencies for -g flag
         if self.validate_dependencies:
+            self.validate_pack_dependencies()
+        if id_set_validations:
+            is_valid, error = id_set_validations.is_unique_file_valid_in_set(self.pack_path, pack_error_ignore_list)
+            if not is_valid:
+                self._add_error(error, self.pack_path)
+
             self.validate_pack_dependencies()
```

Comment:
i think this line can be removed, the `validate_pack_dependencies` is called twice.

Reply:
done, A mistake that happened because of conflicts